test/go/iterative_verifier_test.go

```diff
@@ -1,477 +0,0 @@
-
-	t.InsertCompressedRowInDb(43, testhelpers.TestCompressedData3, t.Ferry.SourceDB)
-	t.InsertCompressedRowInDb(43, testhelpers.TestCompressedData4, t.Ferry.TargetDB)
-
-	result, err := t.verifier.VerifyOnce()
-	t.Require().NotNil(result)
-	t.Require().Nil(err)
-	t.Require().True(result.DataCorrect)
-	t.Require().Equal("", result.Message)
-}
-
-func (t *IterativeVerifierTestSuite) TestBeforeCutoverFailuresFailAgainDuringCutover() {
-	t.InsertRowInDb(42, "foo", t.Ferry.SourceDB)
-	t.InsertRowInDb(42, "bar", t.Ferry.TargetDB)
-
-	err := t.verifier.VerifyBeforeCutover()
-	t.Require().Nil(err)
-
-	result, err := t.verifier.VerifyDuringCutover()
-	t.Require().Nil(err)
-	t.Require().False(result.DataCorrect)
-	t.Require().Equal("verification failed on table: gftest.test_table_1 for pks: 42", result.Message)
-}
-
-func (t *IterativeVerifierTestSuite) TestBeforeCutoverCompressionFailuresFailAgainDuringCutover() {
-	t.InsertCompressedRowInDb(42, testhelpers.TestCompressedData1, t.Ferry.SourceDB)
-	t.InsertCompressedRowInDb(42, testhelpers.TestCompressedData2, t.Ferry.TargetDB)
-
-	err := t.verifier.VerifyBeforeCutover()
-	t.Require().Nil(err)
-
-	result, err := t.verifier.VerifyDuringCutover()
-	t.Require().Nil(err)
-	t.Require().False(result.DataCorrect)
-	t.Require().Equal(fmt.Sprintf("verification failed on table: %s.%s for pks: %s", "gftest", testhelpers.TestCompressedTable1Name, "42"), result.Message)
-}
-
-func (t *IterativeVerifierTestSuite) TestBeforeCutoverDifferentCompressedSameDecompressedDataPassDuringCutover() {
-	t.Require().NotEqual(testhelpers.TestCompressedData3, testhelpers.TestCompressedData4)
-
-	t.InsertCompressedRowInDb(43, testhelpers.TestCompressedData3, t.Ferry.SourceDB)
-	t.InsertCompressedRowInDb(43, testhelpers.TestCompressedData4, t.Ferry.TargetDB)
-
-	err := t.verifier.VerifyBeforeCutover()
-	t.Require().Nil(err)
-
-	result, err := t.verifier.VerifyDuringCutover()
-	t.Require().Nil(err)
-	t.Require().True(result.DataCorrect)
-	t.Require().Equal("", result.Message)
-}
-
-func (t *IterativeVerifierTestSuite) TestErrorsIfMaxDowntimeIsSurpassed() {
-	t.InsertRowInDb(42, "foo", t.Ferry.SourceDB)
-	t.InsertRowInDb(42, "bar", t.Ferry.TargetDB)
-
-	t.verifier.MaxExpectedDowntime = 1 * time.Nanosecond
-	err := t.verifier.VerifyBeforeCutover()
-	t.Require().Regexp("cutover stage verification will not complete within max downtime duration \\(took .*\\)", err.Error())
-}
-
-func (t *IterativeVerifierTestSuite) TestBeforeCutoverFailuresPassDuringCutover() {
-	t.InsertRowInDb(42, "foo", t.Ferry.SourceDB)
-	t.InsertRowInDb(42, "bar", t.Ferry.TargetDB)
-
-	err := t.verifier.VerifyBeforeCutover()
-	t.Require().Nil(err)
-
-	t.UpdateRowInDb(42, "foo", t.Ferry.TargetDB)
-
-	result, err := t.verifier.VerifyDuringCutover()
-	t.Require().Nil(err)
-	t.Require().True(result.DataCorrect)
-	t.Require().Equal("", result.Message)
-}
-
-func (t *IterativeVerifierTestSuite) TestChangingDataChangesHash() {
```

Comment:
There'll be a lot of `GetHashes` test below. All these tests are not applicable anymore because the InlineVerifier computes the hashes on the hashes on the source is obtained with data iteration and the hashes on the target are obtained separately. These test cases are only covered via the overall integration tests. 

I could add some unit tests, but will need to expose some methods as public.